package-lock.json

```diff
@@ -1,3565 +1,8 @@
 {
   "name": "@fullstory/babel-plugin-annotate-react",
   "version": "2.3.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
https://app.circleci.com/projects/github/fullstorydev/fullstory-babel-plugin-annotate-react/config/?branchName=master&pipelineNumber=764

Our CI uses node 12, i built this lock file using node 12 as well which has a lower lockfile version.